_maps/RandomRuins/SpaceRuins/spacehotel.dmm

```diff
@@ -1228,6 +1363,13 @@
 	},
 /turf/template_noop,
 /area/ruin/space/has_grav/hotel)
+"kU" = (
+/obj/structure/cable,
+/obj/structure/rack,
+/obj/item/clothing/suit/space/syndicate/black/blue,
```

Comment:
how about you dont give a space suit thats almost as strong as a syndicate MODsuit on that ruin?
(its defence is:
	melee = 40
	bullet = 50
	laser = 30
	energy = 40
	bomb = 30
	bio = 30
	fire = 80
	acid = 85
	)

Reply:
these suits aren't actually a big deal, there are like, half a dozen of them in random ruins, also they make you slow as shit because they're space suits.